x-pack/plugins/security_solution/package.json

```diff
@@ -13,13 +13,13 @@
     "test:generate": "ts-node --project scripts/endpoint/cli_tsconfig.json scripts/endpoint/resolver_generator.ts"
   },
   "devDependencies": {
-    "@types/md5": "^2.2.0"
-  },
-  "dependencies": {
-    "@types/rbush": "^3.0.0",
-    "@types/seedrandom": ">=2.0.0 <4.0.0",
+    "@types/md5": "^2.2.0",
     "querystring": "^0.2.0",
     "rbush": "^3.0.1",
     "redux-devtools-extension": "^2.13.8"
+  },
+  "dependencies": {
+    "@types/rbush": "^3.0.0",
```

Comment:
These types can't be dev dependencies?

Reply:
Good catch, all `@types/*` can be devDependencies, I'll update my script